### PR TITLE
Feature/blur

### PR DESCRIPTION
**Pull request by veilance**
DOM is fixed so that when a user renders the page, their pins will render dynamically with the HTML generating on the AJAX call (instead of call the EJS file).

The displayPins.ejs file 's HTML for the form has been DELETED (because it's generating on the fly). 
Opening a NEW infowindow or one of your previous infowindows will now CLOSE all infowindows. 

TESTS:
If the user clicks on a pin they own or another location on the map and open an info window, all other info windows CLOSE. 

If they click on a new location on the map, a marker will be created, if they click the marker, an info window opens up and they can enter information (that information isn't saved unless they hit the save button) 

The infowindows now generate on click. There shouldn't be any residuals on the page after creating a new pin. 

TL;DR IT WORKS NOW

### CODE REVIEW

**Comment by zhouism** [comment]
functions conflict resolved
